pkg/storegateway/bucket.go

```diff
@@ -1192,6 +1191,28 @@ func (s *BucketStore) recordSeriesCallResult(safeStats *safeQueryStats) {
 	s.metrics.seriesHashCacheRequests.Add(float64(stats.seriesHashCacheRequests))
 	s.metrics.seriesHashCacheHits.Add(float64(stats.seriesHashCacheHits))
 
+	// Currently fineGrainedChunksCachingEnabled is always false. After it is configurable,
+	// these stats will be recorded by the new caching implementation.
+	if s.fineGrainedChunksCachingEnabled && s.maxSeriesPerBatch > 0 {
+		s.metrics.seriesDataFetched.WithLabelValues("chunks", "normal").Observe(float64(stats.chunksFetched))
```

Comment:
Have you considered renaming "normal" into "fetch"? Basically it's a normal "fetch".

Reply:
[nit] Also, does it make sense to call it "fetched" and "refetched", given we have "parsed" and "selected"?